### DISCUSSION TITLE
scheduling workflows

### DISCUSSION THREAD

**Discussion by trasferimento** [AutomaApp/automa#1026] [Q&A]
Hi,
thanks for the wonderful job you are doing with this Extensions.
About managing scheduled workflows,
- how to delete one workflow from the scheduling view, when already scheduled ?
- how to deactivate a single schedule ( there is a flag = active.. so it seems it could be deactivated as well.. )?

Thks

**Comment by Kholid060** [comment]
- You can delete the schedule by opening the workflow and then editing the trigger block 
- It's not possible to deactivate the schedule, you can only delete the schedule

**Reply by wohenbushuang** [reply]
What if I want the workflow to stop when I am in a vocation (not fixed date like weekend), and activate again after I back to work, do I need to edit all the trigger blocks in the workflows? thanks.

**Comment by trasferimento** [comment]
Ok,
Thanks. It is now clear!

Il Ven 23 Dic 2022, 04:07 Ahmad Kholid ***@***.***> ha
scritto:
